### PR TITLE
Translated missing dutch sentences

### PR DESCRIPTION
**Pull request by mjarkk**
@glvr182 Can you take a look over the translations there are some translations that might have a better alternative translation.  

@jesseduffield I've removed `Damn` because the closes word i could think of in dutch is `Fuck` or `Shit`. I don't this those are really appropriate words, there are some more nice alternative sentences but those seems a bit childish like `ow nee`, `verdraait` of `oeps`.

### CODE REVIEW

File: docs/keybindings/Keybindings_nl.md [hunk 1]
```diff
@@ -57,55 +57,55 @@
...
-  <kbd>d</kbd>: delete commit
-  <kbd>J</kbd>: move commit down one
-  <kbd>K</kbd>: move commit up one
-  <kbd>e</kbd>: edit commit
-  <kbd>A</kbd>: amend commit with staged changes
+  <kbd>d</kbd>: verwijder commit
+  <kbd>J</kbd>: verplaats commit 1 omlaag
+  <kbd>K</kbd>: verplaats commit 1 omhoog
+  <kbd>e</kbd>: wijzig commit
+  <kbd>A</kbd>: wijzig commit met gestagte veranderingen
```
Comment (glvr182) [inline docs/keybindings/Keybindings_nl.md]: Maybe "Wijzig commit met opgeslagen verandering" would be better.

File: docs/keybindings/Keybindings_nl.md [hunk 2]
```diff
@@ -57,55 +57,55 @@
...
   <kbd>p</kbd>: pick commit (when mid-rebase)
-  <kbd>t</kbd>: revert commit
-  <kbd>c</kbd>: copy commit (cherry-pick)
-  <kbd>C</kbd>: copy commit range (cherry-pick)
-  <kbd>v</kbd>: paste commits (cherry-pick)
-  <kbd>enter</kbd>: view commit's files
-  <kbd>space</kbd>: diff specific commits
+  <kbd>t</kbd>: zet commit terug
```
Comment (glvr182) [inline docs/keybindings/Keybindings_nl.md]: "Maak commit ongedaan"

File: docs/keybindings/Keybindings_nl.md [hunk 3]
```diff
@@ -57,55 +57,55 @@
...
+  <kbd>c</kbd>: kopiëer commit (cherry-pick)
+  <kbd>C</kbd>: kopiëer commit range (cherry-pick)
+  <kbd>v</kbd>: plak commits (cherry-pick)
+  <kbd>enter</kbd>: bekijk commited bestanden
+  <kbd>space</kbd>: diff specifieke commit's
```
Comment (glvr182) [inline docs/keybindings/Keybindings_nl.md]: "Bekijk verschillen tussen specifieke commits"

File: docs/keybindings/Keybindings_nl.md [hunk 4]
```diff
@@ -57,55 +57,55 @@
...
 
 ## Commit files
 
 <pre>
-  <kbd>esc</kbd>: go back
-  <kbd>c</kbd>: checkout file
-  <kbd>d</kbd>: discard this commit's changes to this file
+  <kbd>esc</kbd>: ga terug
+  <kbd>c</kbd>: checkout bestand
+  <kbd>d</kbd>: uitsluit deze commit zijn veranderingen aan dit bestand
```
Comment (glvr182) [inline docs/keybindings/Keybindings_nl.md]: "Verwijder wijzigingen van dit bestand"

File: docs/keybindings/Keybindings_nl.md [hunk 5]
```diff
@@ -57,55 +57,55 @@
...
   <kbd>o</kbd>: open bestand
 </pre>
 
 ## Main (Normal)
 
 <pre>
-  <kbd>PgDn</kbd>: scroll down
-  <kbd>PgUp</kbd>: scroll up
+  <kbd>PgDn</kbd>: scroll omhoog
+  <kbd>PgUp</kbd>: scroll omlaag
```
Comment (glvr182) [inline docs/keybindings/Keybindings_nl.md]: I think you mixed these two up :smile:

File: docs/keybindings/Keybindings_nl.md [hunk 6]
```diff
@@ -115,10 +115,10 @@
 <pre>
   <kbd>esc</kbd>: ga terug naar het bestanden paneel
   <kbd>space</kbd>: pick hunk
-  <kbd>b</kbd>: pick both hunks
-  <kbd>◄</kbd>: select previous conflict
-  <kbd>►</kbd>: select next conflict
-  <kbd>▲</kbd>: select top hunk
-  <kbd>▼</kbd>: select bottom hunk
-  <kbd>z</kbd>: undo
+  <kbd>b</kbd>: pick bijde hunks
```
Comment (glvr182) [inline docs/keybindings/Keybindings_nl.md]: "beide"

File: pkg/i18n/dutch.go [hunk 7]
```diff
@@ -453,34 +453,34 @@ func addDutch(i18nObject *i18n.Bundle) error {
 			Other: "Merging",
 		}, &i18n.Message{
 			ID:    "ConfirmRebase",
-			Other: "Are you sure you want to rebase {{.checkedOutBranch}} onto {{.selectedBranch}}?",
+			Other: "Weet je zeker dat je {{.checkedOutBranch}} naar {{.selectedBranch}} wil rebasen?",
```
Comment (glvr182) [inline pkg/i18n/dutch.go]: "naar" = "op"

File: pkg/i18n/dutch.go [hunk 8]
```diff
@@ -453,34 +453,34 @@ func addDutch(i18nObject *i18n.Bundle) error {
...
 		}, &i18n.Message{
 			ID:    "ConfirmMerge",
-			Other: "Are you sure you want to merge {{.selectedBranch}} into {{.checkedOutBranch}}?",
+			Other: "Weet je zeker dat je {{.selectedBranch}} in {{.checkedOutBranch}} wil mergen?",
 		}, &i18n.Message{
 			ID:    "FwdNoUpstream",
-			Other: "Cannot fast-forward a branch with no upstream",
+			Other: "can niet de branch niet vooruitspoelen met upstream",
```
Comment (glvr182) [inline pkg/i18n/dutch.go]: "Kan niet de branch vooruitspoelen zonder upstream"

File: pkg/i18n/dutch.go [hunk 9]
```diff
@@ -453,34 +453,34 @@ func addDutch(i18nObject *i18n.Bundle) error {
...
 		}, &i18n.Message{
 			ID:    "ErrorOccurred",
-			Other: "An error occurred! Please create an issue at https://github.com/jesseduffield/lazygit/issues",
+			Other: "Er is iets fout gegaan! Zou je een issue willen maken hier https://github.com/jesseduffield/lazygit/issues",
```
Comment (glvr182) [inline pkg/i18n/dutch.go]: "Er is iets fout gegaan! Zou je hier een issue aan willen maken:"

File: pkg/i18n/dutch.go [hunk 10]
```diff
@@ -453,34 +453,34 @@ func addDutch(i18nObject *i18n.Bundle) error {
...
 		}, &i18n.Message{
 			ID:    "FwdCommitsToPush",
-			Other: "Cannot fast-forward a branch with commits to push",
+			Other: "Kan niet vooruitspoelen, er zijn commits not niet gepushed",
```
Comment (glvr182) [inline pkg/i18n/dutch.go]: "Je kan niet vooruitspoelen als de branch geen nieuwe commits heeft"

File: pkg/i18n/dutch.go [hunk 11]
```diff
@@ -453,34 +453,34 @@ func addDutch(i18nObject *i18n.Bundle) error {
...
-			Other: "soft reset",
+			Other: "zacht reset",
 		}, &i18n.Message{
 			ID:    "CantRebaseOntoSelf",
-			Other: "You cannot rebase a branch onto itself",
+			Other: "Je kan niet een branch rebasen op zichzelf",
 		}, &i18n.Message{
 			ID:    "SureSquashThisCommit",
-			Other: "Are you sure you want to squash this commit into the commit below?",
+			Other: "Weet je zeker dat je deze commit wil squashen in de commit hieronder?",
```
Comment (glvr182) [inline pkg/i18n/dutch.go]: "Squashen" can be changed to "samenvoegen met de commit hieronder"

File: pkg/i18n/dutch.go [hunk 12]
```diff
@@ -489,130 +489,130 @@ func addDutch(i18nObject *i18n.Bundle) error {
 			Other: "pick commit (when mid-rebase)",
 		}, &i18n.Message{
 			ID:    "revertCommit",
-			Other: "revert commit",
+			Other: "terug naar commit",
```
Comment (glvr182) [inline pkg/i18n/dutch.go]: "Commit ongedaan maken"

File: pkg/i18n/dutch.go [hunk 13]
```diff
@@ -489,130 +489,130 @@ func addDutch(i18nObject *i18n.Bundle) error {
...
 			ID:    "Undo",
-			Other: "undo",
+			Other: "ongedaan maken",
 		}, &i18n.Message{
 			ID:    "PickHunk",
 			Other: "pick hunk",
 		}, &i18n.Message{
 			ID:    "PickBothHunks",
-			Other: "pick both hunks",
+			Other: "pick bijde hunks",
```
Comment (glvr182) [inline pkg/i18n/dutch.go]: "beide"

File: pkg/i18n/dutch.go [hunk 14]
```diff
@@ -489,130 +489,130 @@ func addDutch(i18nObject *i18n.Bundle) error {
...
-			Other: "Merge Options",
+			Other: "Merge Opties",
 		}, &i18n.Message{
 			ID:    "RebaseOptionsTitle",
-			Other: "Rebase Options",
+			Other: "Rebase Opties",
 		}, &i18n.Message{
 			ID:    "ConflictsResolved",
-			Other: "all merge conflicts resolved. Continue?",
+			Other: "alle merge conflicten zijn opgelost. Verder?",
```
Comment (glvr182) [inline pkg/i18n/dutch.go]: "Wilt u verder gaan?"

File: pkg/i18n/dutch.go [hunk 15]
```diff
@@ -489,130 +489,130 @@ func addDutch(i18nObject *i18n.Bundle) error {
...
-			Other: "Not enough room",
+			Other: "Niet genoeg ruimte",
 		}, &i18n.Message{
 			ID:    "YouAreHere",
-			Other: "YOU ARE HERE",
+			Other: "JE BENT HIER",
 		}, &i18n.Message{
 			ID:    "rewordNotSupported",
-			Other: "rewording commits while interactively rebasing is not currently supported",
+			Other: "herformatteren van commits in interactiev rebasen is nog niet gesupport",
```
Comment (glvr182) [inline pkg/i18n/dutch.go]: "interactief"
and "gesupport" can be changed to "ondersteund"

File: pkg/i18n/dutch.go [hunk 16]
```diff
@@ -489,130 +489,130 @@ func addDutch(i18nObject *i18n.Bundle) error {
...
 			ID:    "SureCherryPick",
-			Other: "Are you sure you want to cherry-pick the copied commits onto this branch?",
+			Other: "Weet je zeker dat je de gekopieerde commits naar deze branch wil cherry-picken?",
 		}, &i18n.Message{
 			ID:    "CherryPick",
 			Other: "Cherry-Pick",
 		}, &i18n.Message{
 			ID:    "CannotRebaseOntoFirstCommit",
-			Other: "You cannot interactive rebase onto the first commit",
+			Other: "Je kan niet interactiev rebasen naar de eerste commit",
```
Comment (glvr182) [inline pkg/i18n/dutch.go]: "interactief"

File: pkg/i18n/dutch.go [hunk 17]
```diff
@@ -489,130 +489,130 @@ func addDutch(i18nObject *i18n.Bundle) error {
...
-			Other: "select next conflict",
+			Other: "selecteer volgende conflict",
 		}, &i18n.Message{
 			ID:    "SelectTop",
-			Other: "select top hunk",
+			Other: "selecteer bovenste hunk",
 		}, &i18n.Message{
 			ID:    "SelectBottom",
-			Other: "select bottom hunk",
+			Other: "selecteer bodem hunk",
```
Comment (glvr182) [inline pkg/i18n/dutch.go]: "onderste"

File: pkg/i18n/dutch.go [hunk 18]
```diff
@@ -621,118 +621,118 @@ func addDutch(i18nObject *i18n.Bundle) error {
...
 			ID:    "CreateRepo",
-			Other: "Not in a git repository. Create a new git repository? (y/n): ",
+			Other: "Niet in een git repository. Creëer een nieuwe git repository? (y/n): ",
 		}, &i18n.Message{
 			ID:    "AutoStashTitle",
 			Other: "Autostash?",
 		}, &i18n.Message{
 			ID:    "AutoStashPrompt",
-			Other: "You must stash and pop your changes to bring them across. Do this automatically? (enter/esc)",
+			Other: "Je moet je veranderingen stashen en poppen om ze over te bregen. Dit automaties doen? (enter/esc)",
```
Comment (glvr182) [inline pkg/i18n/dutch.go]: "automatisch"

Comment (mjarkk) [reply pkg/i18n/dutch.go]: Thx fixed it

File: pkg/i18n/dutch.go (hunk 17 above)
Comment (mjarkk) [reply pkg/i18n/dutch.go]: Thx fixed it

**Comment by mjarkk** [comment]
@glvr182 thanks for the review, Fixed them

**Comment by jesseduffield** [comment]
thanks for doing this! looks good to me

**Pull request merged**